docs/articles/Resources/Overview.md

```diff
@@ -3,17 +3,18 @@ uid: ResourceConcept
 ---
 # Resource concept
 
-A resource is everything needed to produce a product. It can be a physical device like a Station or a Cell and it can be also something non physical like a SerialNumberProvider that calculates a new serial number.
+A resource is everything needed to produce a product. 
```

Comment:
Since the AL has evolved beyond the control system, we should extend this section with a more general definition. Resources represent physical assets and logical objects like drivers within a cyber physical system. They can be coupled with active devices or represents passive objects like a table.